src/thor/dijkstras.cc

```diff
@@ -39,20 +39,12 @@ uint32_t GetOperatorId(const GraphTile* tile,
 namespace valhalla {
 namespace thor {
 
-constexpr uint32_t kBucketCount = 20000;
-constexpr uint32_t kInitialEdgeLabelCount = 500000;
```

Comment:
this was hardcoded for isochrones but now that its used by reach checks we needed to refactor it to be something the derived-class implementor must provide